submissions/OwoeyeGbenga/Php_Task/Public/create.php

```diff
@@ -0,0 +1,60 @@
+<?php
+if (isset($_POST['submit'])) {
+    require "config.php";
+    require "helper.php";
+
+    try {
+        $connection = new PDO($dsn, $username, $password, $options);
+
+        $newProduct = array(
+            "itemname" => $_POST['itemname'],
+            "quantity" => $_POST['quantity'],
+            "brand_id" => $_POST['brand'] + 1
+        );
+
+        /**
+         * This is equivalent to
+         * INSERT INTO products (itemname, quantity) values (:itemname, :quantity)
+         */
+        $sql = sprintf(
+            "INSERT INTO %s (%s) values (%s)",
+            "products",
+            implode(", ", array_keys($newProduct)),
+            ":" . implode(", :", array_keys($newProduct))
+        );
+
+        $statement = $connection->prepare($sql);
+        $statement->execute($newProduct);
+    } catch (PDOException $error) {
+        echo $sql . "<br>" . $error->getMessage();
+    }
+}
+?>
+
+<?php if (isset($_POST['submit']) && $statement) { ?>
+    > <?php echo $_POST['itemname']; ?> successfully added.
+<?php } ?>
+
+
+<h2>Add A Product </h2>
+
+<form method="post">
+    <label for="itemname">Name of Product</label>
+    <input type="text" name="itemname" id="itemname">
+    <?php session_start(); ?>
```

Comment:
Hello @olugbengs12 . This means that if I load this file without loading `fetch.php` first, you will not have any brands. Please refrain from doing this and find alternative ways.
For instance, you can query your database in this file similar to what is done in `overview.php

Reply:
Hello @olugbengs12. I have gone over this pull request and the above comment is still left unattended to.